shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -0,0 +1,234 @@
+      return 0;
+    }
+    ReportWorkerOptions options = getOptions(cl);
+    printWorkerCapacityInfo(options);
+    return 0;
+  }
+
+  /**
+   * Prints Alluxio capacity information.
+   *
+   * @param options ReportWorkerOptions to define the report worker info range
+   */
+  public void printWorkerCapacityInfo(ReportWorkerOptions options) throws IOException {
+    List<ReportWorkerInfo> workerInfolist = mBlockMasterClient.getReportWorkerInfoList(options);
+
+    // Summarize the worker capacity information
+    long sumCapacityBytes = 0;
+    long sumUsedBytes = 0;
+    Map<String, Long> sumTotalBytesOnTiersMap = initializeTierMap();
+    Map<String, Long> sumUsedBytesOnTiersMap = initializeTierMap();
+
+    // Cache the worker capacity information
+
+    for (ReportWorkerInfo workerInfo : workerInfolist) {
+      mStringBuilder.append(String.format("%-20s %-20s %-20s %-20s %-20s%n",
+          "Node Name", "Last Heartbeat", "Workers Capacity", "Space Used", "Used Percentage"));
+
+      long usedBytes = workerInfo.getUsedBytes();
+      long capacityBytes = workerInfo.getCapacityBytes();
+      String usedPercentageInfo = "Unavailable";
+      if (capacityBytes != 0) {
+        int usedPercentage = (int) (100L * usedBytes / capacityBytes);
+        usedPercentageInfo = String.format("%s%%", usedPercentage);
+      }
+      mStringBuilder.append(String.format("%-20s %-20s %-20s %-20s %-20s%n",
+          workerInfo.getAddress().getHost(),
+          workerInfo.getLastContactSec(),
+          FormatUtils.getSizeFromBytes(capacityBytes),
+          FormatUtils.getSizeFromBytes(usedBytes),
+          usedPercentageInfo));
+
+      sumCapacityBytes += capacityBytes;
+      sumUsedBytes += usedBytes;
+
+      Map<String, Long> totalBytesOnTiers = workerInfo.getCapacityBytesOnTiers();
+      if (totalBytesOnTiers != null) {
+        for (Map.Entry<String, Long> totalBytesTier : totalBytesOnTiers.entrySet()) {
+          String tier = totalBytesTier.getKey();
+          long value = totalBytesTier.getValue();
+          sumTotalBytesOnTiersMap.put(tier, value + sumTotalBytesOnTiersMap.get(tier));
+        }
+      }
+
+      Map<String, Long> usedBytesOnTiers = workerInfo.getUsedBytesOnTiers();
+      if (usedBytesOnTiers != null) {
+        for (Map.Entry<String, Long> usedBytesTier: usedBytesOnTiers.entrySet()) {
+          String tier = usedBytesTier.getKey();
+          long value = usedBytesTier.getValue();
+          sumUsedBytesOnTiersMap.put(tier, value + sumUsedBytesOnTiersMap.get(tier));
+        }
+      }
+    }
+
+    // Print summarized worker capacity information
+    if (options.getWorkerRange().equals(WorkerRange.SPECIFIED)
+        && sumCapacityBytes + sumUsedBytes == 0) {
+      System.out.println(USAGE);
+      throw new InvalidArgumentException("Worker IP addresses are invalid.");
+    }
+
+    mIndentationLevel = 0;
+    print(String.format("Capacity Information for %s Workers: ", options.getWorkerRange()));
```

Comment:
is this going to print the worker range with all uppercase letters? I don't think we want that?